app/assets/javascripts/discourse/app/components/modal/avatar-selector.hbs

```diff
@@ -0,0 +1,137 @@
+<DModal
+  @bodyClass="avatar-selector"
+  @closeModal={{@closeModal}}
+  @title={{i18n "user.change_avatar.title"}}
+  class="avatar-selector-modal"
+>
+  <:body>
+    {{#if this.showSelectableAvatars}}
+      <div class="selectable-avatars">
+        {{#each this.selectableAvatars as |avatar|}}
+          <a
+            href
+            class="selectable-avatar"
+            {{on "click" (fn this.selectAvatar avatar)}}
+          >
+            {{bound-avatar-template avatar "huge"}}
+          </a>
+        {{/each}}
+      </div>
+      {{#if this.showAvatarUploader}}
+        <h4>{{html-safe (i18n "user.change_avatar.use_custom")}}</h4>
+      {{/if}}
+    {{/if}}
+    {{#if this.showAvatarUploader}}
+      {{#if this.user.use_logo_small_as_avatar}}
+        <div class="avatar-choice">
+          <RadioButton
+            @id="logo-small"
+            @name="logo"
+            @value="logo"
+            @selection={{this.selected}}
+            @onChange={{this.onSelectedChanged}}
+          />
+          <label class="radio" for="logo-small">{{bound-avatar-template
+              this.siteSettings.site_logo_small_url
+              "large"
+            }}
+            {{html-safe (i18n "user.change_avatar.logo_small")}}</label>
```

Comment:
`yarn lint:fix` is fine and does what it's supposed to, but is a bit too conservative and the code needs to be massaged a bit 😉 

though rather than calling `yarn lint:fix` manually I strongly recommend enabling linting integrations in your code editor (so it would format on save)